drivers/rtc/rtc_emul.c

```diff
@@ -10,13 +10,6 @@
 #include <zephyr/device.h>
 #include <zephyr/drivers/rtc.h>
 
-struct rtc_emul_data;
```

Comment:
This change us better achieved by using the CONTAINER_OF macro, which removes the need for this structure, while keeping the work queue item.

Reply:
Obsolete: change was reverted.